components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -178,6 +178,14 @@
     public static final String GOVERNANCE_COMPONENT_REGISTRY_LOCATION = "/repository/components/org.wso2.carbon" +
             ".governance";
 
+    // registry location for OpenAPI files
+    public static final String OPENAPI_ARCHIVES_TEMP_FOLDER = "OPENAPI-archives";
+    public static final String OPENAPI_EXTRACTED_DIRECTORY = "extracted";
+    public static final String OPENAPI_ARCHIVE_ZIP_FILE = "openapi-archive.zip";
+    public static final String CHARSET = "UTF-8";
```

Comment:
probably you can use existing Constance from the Charset class.

Reply:
https://github.com/wso2/carbon-apimgt/pull/8509/commits/9ded714b1b9db9c1e33fa3fef5239858920580f9#diff-d28b77ae984cf341809858ec0fb93f3fL185